src/test/java/org/folio/circulation/support/http/client/VertxWebClientOkapiHttpClientTests.java

```diff
@@ -50,7 +52,7 @@
   private static VertxAssistant vertxAssistant;
 
   @Rule
-  public WireMockRule fakeWebServer = new WireMockRule();
+  public WireMockRule fakeWebServer = new WireMockRule(options().port(8081));
```

Comment:
Thank you for discovering this, I think this was catching out my attempt to do this work.

How did you discover this was the issue?

I wonder why a Vert.x upgrade causes this. Maybe it runs something on port 8080 by default.

Reply:
It generated an error message I'd seen before and associated with port binding problems-I can't remember the exact text, but it was something like. "cannot bind, collision" or suchlike.  I don't think this is connected to the upgrade-this was happening in master, *before* I implemented the upgrades.  I make a habit of always running tests on master branch before I do any work to catch just this sort of thing-I had to fix this *before* I started the upgrade.

Reply:
> I make a habit of always running tests on master branch before I do any work to catch just this sort of thing-I had to fix this _before_ I started the upgrade.

That's a good habit.